layouts/project/single.html

```diff
@@ -2,29 +2,25 @@
 	<div class="margin-bottom-30">
 		{{ .Content }}
 	</div>
-	{{ if gt (len .Params.eclipse_projects) 0 }}
-	<h2 class="margin-bottom-40">Eclipse projects</h2>
-	<div class="row">
-	{{ range .Params.eclipse_projects }}
-		{{range first 1 (where $.Site.Data.eclipsefdn_projects "project_id" .) }}
-		<div class="col-sm-12 margin-bottom-30 match-height-item-by-row flex-column project-item">
-		  <div class="padding-bottom-20 text-center background-white flex-grow">
-		    <div >
-		      <h3>
-		        <a href="{{ .url }}">
-			      <img class="img img-responsive margin-auto padding-bottom-20 " src="{{ .logo }}" title="{{ .name }} logo"/>
-				  {{ .name }}
-			    </a>
-			  </h3>
+	{{ if .Params.referenced_eclipse_projects }}
+		{{ if gt (len .Params.referenced_eclipse_projects) 0 }}
+			<h2 class="margin-top-30 margin-bottom-20">Referenced Eclipse projects</h2>
+			<div class="row">
```

Comment:
If you use a row, there should be a wrapping col around all of these cols. With the current look and feel, the boxes are countersunk 15px left (due to the row) and on screens > 1000px width, the box is against the edge of the screen.

The solution here is to likely remove the row (as it isn't needed here) or wrap the entire "Eclipse Projects" section in a col-xs-24 div.

Reply:
The reason that the boxes are too far countersunk is that there are 2 rows in a row without a column class between them

Reply:
removed row